### PR TITLE
SUS-2151: make CoppaTool and Wall IPv6 compliant

### PR DESCRIPTION
**Pull request by ghost**
https://wikia-inc.atlassian.net/browse/SUS-2151

There is a few places using `0.0.0.0` as non-routable IP address. We are migrating to IPv6 so I have changed those to `::`. It originally started with the CoppaTool and the change to Wall is an extra mile based on `grep` output. :-)

These two cases have nothing to do with networking and/or querying some database using things like subnet masks. These are justs strings. The CoppaTool changes actual IP in logs for privacy protection. The Wall - well, it's Wall and it has its edgecases and undefined behaviours, not to be confused with AI or any kind of identity :-)

@ljagiello @macbre @TK-999

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 62 issues
* ![MAJOR][MAJOR] 43 major
* ![MINOR][MINOR] 8 minor
* ![INFO][INFO] 11 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [CoppaToolController.class.php#L16](https://github.com/Wikia/app/blob/df3426e09526e183572016fd61c0095c42824bd0/extensions/wikia/CoppaTool/CoppaToolController.class.php#L16): Change the output format of this i18n string away from plain(). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3APlainFormatWfMessage)
1. ![MAJOR][MAJOR] [CoppaToolController.class.php#L25](https://github.com/Wikia/app/blob/df3426e09526e183572016fd61c0095c42824bd0/extensions/wikia/CoppaTool/CoppaToolController.class.php#L25): Change the output format of this i18n string away from plain(). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3APlainFormatWfMessage)
1. ![MAJOR][MAJOR] [CoppaToolController.class.php#L32](https://github.com/Wikia/app/blob/df3426e09526e183572016fd61c0095c42824bd0/extensions/wikia/CoppaTool/CoppaToolController.class.php#L32): Change the output format of this i18n string away from plain(). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3APlainFormatWfMessage)
1. ![MAJOR][MAJOR] [CoppaToolController.class.php#L55](https://github.com/Wikia/app/blob/df3426e09526e183572016fd61c0095c42824bd0/extensions/wikia/CoppaTool/CoppaToolController.class.php#L55): Change the output format of this i18n string away from plain(). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3APlainFormatWfMessage)
1. ![MAJOR][MAJOR] [CoppaToolController.class.php#L64](https://github.com/Wikia/app/blob/df3426e09526e183572016fd61c0095c42824bd0/extensions/wikia/CoppaTool/CoppaToolController.class.php#L64): Change the output format of this i18n string away from plain(). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3APlainFormatWfMessage)
1. ![MAJOR][MAJOR] [CoppaToolController.class.php#L86](https://github.com/Wikia/app/blob/df3426e09526e183572016fd61c0095c42824bd0/extensions/wikia/CoppaTool/CoppaToolController.class.php#L86): Change the output format of this i18n string away from plain(). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3APlainFormatWfMessage)
1. ![MAJOR][MAJOR] [CoppaToolController.class.php#L95](https://github.com/Wikia/app/blob/df3426e09526e183572016fd61c0095c42824bd0/extensions/wikia/CoppaTool/CoppaToolController.class.php#L95): Change the output format of this i18n string away from plain(). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3APlainFormatWfMessage)
1. ![MAJOR][MAJOR] [CoppaToolController.class.php#L105](https://github.com/Wikia/app/blob/df3426e09526e183572016fd61c0095c42824bd0/extensions/wikia/CoppaTool/CoppaToolController.class.php#L105): Change the output format of this i18n string away from plain(). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3APlainFormatWfMessage)
1. ![MAJOR][MAJOR] [CoppaToolController.class.php#L109](https://github.com/Wikia/app/blob/df3426e09526e183572016fd61c0095c42824bd0/extensions/wikia/CoppaTool/CoppaToolController.class.php#L109): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
1. ![MAJOR][MAJOR] [CoppaToolController.class.php#L110](https://github.com/Wikia/app/blob/df3426e09526e183572016fd61c0095c42824bd0/extensions/wikia/CoppaTool/CoppaToolController.class.php#L110): Refactor this code to rely on RequestContext instead of context-dependent globals. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: extensions/wikia/CoppaTool/CoppaToolController.class.php [hunk 1]
```diff
@@ -139,7 +139,7 @@ public function renameIPAddress() {
 		}
 
 		$ipAddr = IP::sanitizeIP( $ipAddr );
-		$newIpAddr = '0.0.0.0';
+		$newIpAddr = '::';
```
Comment (macbre) [inline extensions/wikia/CoppaTool/CoppaToolController.class.php]: What if we introduce a constant instead? Say `Wikia::NOT_ROUTABLE_IP`

Comment (ghost) [reply extensions/wikia/CoppaTool/CoppaToolController.class.php]: Yeah, that's a good idea. Named constant says what it means. I'll change it. Thanks.

**Comment by ghost** [comment]
@macbre — I've decided to keep those named constants together with the rest of IP-related stuff. Please, review.

**Review by macbre — APPROVED** [review]

**Pull request merged**